signinwithapplebutton/src/main/java/com/willowtreeapps/signinwithapplebutton/SignInWithAppleService.kt

```diff
@@ -71,7 +70,7 @@ class SignInWithAppleService(
                 val authenticationUri = Uri
                     .parse("https://appleid.apple.com/auth/authorize")
                     .buildUpon().apply {
-                        appendQueryParameter("response_type", "code")
+                        appendQueryParameter("response_type", "code id_token")
```

Comment:
Not sure what's going on here.
Should be `code id_token` or `code_id_token` or `code ` + `idTokenValue`?

Reply:
`code id_token` is correct!